### PR TITLE
[LLD][COFF] Process all ARM64EC import symbols in MapFile's getSymbols

### PR DESCRIPTION
**Pull request by cjacek**

### CODE REVIEW

File: lld/COFF/Chunks.h [hunk 1]
```diff
@@ -624,7 +624,8 @@ class ImportThunkChunkARM64EC : public ImportThunkChunk {
   MachineTypes getMachine() const override { return ARM64EC; }
   void writeTo(uint8_t *buf) const override;
 
-  Defined *exitThunk;
+  Defined *exitThunk = nullptr;
```
Comment (mstorsjo) [inline lld/COFF/Chunks.h:627]: I guess the null initialization of `exitThunk` is an unrelated change? Although probably good in itself.

**Review by mstorsjo — APPROVED** [review]
Looks good overall, but please mention MapFile in the subject somewhere here as well.

File: lld/COFF/Chunks.h (hunk 1 above)
Comment (cjacek) [reply lld/COFF/Chunks.h:627]: Yes, I will split it.
